packages/vow/test/test-watch.js

```diff
@@ -50,6 +68,28 @@ const runTests = async t => {
   resolver3.reject(Error('disco2'));
   resolver3.resolve(vow2);
   t.is(await when(watch(connVow3P, makeAckWatcher(packet))), 'rejected');
-};
+});
+
+test('promise watcher', async t => {
+  const zone = makeHeapZone();
+  const { watch, when } = prepareVowTools(zone, {
+    rejectionMeansRetry: reason => reason === 'disconnected',
+  });
+
+  const pk = makePromiseKit();
+  pk.resolver.reject('disconnected');
+
+  const vow = watch(pk.promise, {
```

Comment:
Let's please not define a `makePromiseKit` function with a different signature than the [same-named function in endo](https://github.com/endojs/endo/blob/a4a5a75a4200b471fb972061179d9c0b3b0c9f32/packages/promise-kit/index.js#L15-L29), even if only for testing. Another name would be fine, but my preference would be to just use the endo one.
```suggestion
  const { promise, reject } = makePromiseKit();
  reject('disconnected');

  const vow = watch(promise, {
```

Reply:
Agreed.